### PR TITLE
Bug 1663980 - v18.0.0 of webcompat system addon

### PR DESCRIPTION
**Pull request by ksy36**
r? @denschub

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -5999,8 +5998,7 @@
           "version": "5.1.2",
           "resolved": "https://registry.npmjs.org/safe-buffer/-/safe-buffer-5.1.2.tgz",
           "integrity": "sha512-Gd2UZBJDkXlY7GbJxfsE8/nvKkUEU1G38c1siN6QP6a9PT9MmHB8GnpscSmMJSoF8LOIrt8ud/wPtojys4G6+g==",
-          "dev": true,
-          "optional": true
+          "dev": true
         },
         "string_decoder": {
```
Comment (denschub) [inline package-lock.json:6003]: Huh, is this something that always happens to you? I don't think it should be in this patch, unless that can't be avoided. :)

**Review by denschub — COMMENTED** [review]
Looking good, one comment though!

File: package-lock.json (hunk 1 above)
Comment (ksy36) [reply package-lock.json:6004]: Hmm, no I think it's the first time :) I've removed it

**Comment by ksy36** [comment]
Thanks!

**Review by denschub — APPROVED** [review]

**Pull request merged**